django/http/response.py

```diff
@@ -436,15 +436,15 @@ def set_headers(self, filelike):
             else:
                 self['Content-Type'] = 'application/octet-stream'
 
-        if self.as_attachment:
-            filename = self.filename or os.path.basename(filename)
-            if filename:
-                try:
-                    filename.encode('ascii')
-                    file_expr = 'filename="{}"'.format(filename)
-                except UnicodeEncodeError:
-                    file_expr = "filename*=utf-8''{}".format(quote(filename))
-                self['Content-Disposition'] = 'attachment; {}'.format(file_expr)
+        filename = self.filename or os.path.basename(filename)
+        if filename:
+            disposition = 'attachment' if self.as_attachment else 'inline'
+            try:
+                filename.encode('ascii')
+                file_expr = f'filename="{filename}"'
+            except UnicodeEncodeError:
+                file_expr = f"filename*=utf-8''{quote(filename)}"
```

Comment:
Stick to `.format` for now. Until we settle to switch to f strings for the whole code base we should stick one format.

Reply:
Alright, done.
Didn't think it'll be an issue given that only ~10 lines below there's still %-formatting used... :)

Reply:
yeah `.format` crept in while we were supposed to stick to `%`. anyway lets make sure we don't have three ways of formatting strings across the code base 😅 .